test/conformance/memory/memory_adapter_hip.match

```diff
@@ -1,2 +1,20 @@
-urMemBufferCreateWithNativeHandleTest.Success/AMD_HIP_BACKEND___{{.*}}_
-Segmentation fault
+urMemGetInfoTest.Success/AMD_HIP_BACKEND___{{.*}}
+urMemGetInfoTest.InvalidNullPointerParamValue/AMD_HIP_BACKEND___{{.*}}
+urMemGetInfoTest.InvalidNullPointerParamValue/AMD_HIP_BACKEND___{{.*}}
+urMemGetInfoTest.InvalidNullPointerPropSizeRet/AMD_HIP_BACKEND___{{.*}}
+urMemGetInfoTest.InvalidNullPointerPropSizeRet/AMD_HIP_BACKEND___{{.*}}
+urMemImageCreateTest.InvalidSize/AMD_HIP_BACKEND___{{.*}}
+urMemImageGetInfoTest.Success/AMD_HIP_BACKEND___{{.*}}
```

Comment:
These `MemImageGetInfo`s are unsupported

Reply:
Sounds like this test needs to allow these being unsupported. I can look into that.

Reply:
OK great. Will I change this `.match` or just add a TODO?

Reply:
I expect it'll be a separate PR so this line will need to stay in the `.match` file in order to pass testing. I don't actually know if these `.match` files support comments.